src/chart/cartesianChart.js

```diff
@@ -0,0 +1,233 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.chart.cartesianChart = function(xScale, yScale) {
+
+        xScale = xScale || d3.scale.linear();
+        yScale = yScale || d3.scale.linear();
+
+        var xAxisHeight = 20,
+            yAxisWidth = 40,
+            yAxisLabelWidth = 20,
+            xAxisLabelHeight = 20,
+            chartLabelHeight = 20,
+            yAxisLabel = 'y-axis',
+            xAxisLabel = 'x-axis',
+            chartLabel = 'Chart',
+            plotArea = fc.series.line(),
+            decorate = fc.util.fn.noop;
+
+        var xAxis = fc.svg.axis()
+            .scale(xScale)
+            .orient('bottom');
+
+        var yAxis = fc.svg.axis()
+            .scale(yScale)
+            .orient('right');
+
+        var containerDataJoin = fc.util.dataJoin()
+            .selector('svg.cartesian-chart')
+            .element('svg')
+            .attr({'class': 'cartesian-chart', 'layout-css': 'flex: 1'});
+
+        var cartesianChart = function(selection) {
+
+            selection.each(function(data, index) {
+
+                var container = d3.select(this);
+
+                var svg = containerDataJoin(container, [data]);
+                svg.enter().html(
+                    '<g class="title"> \
+                        <text class="label" layout-css="height: 0; width: 0"/> \
+                    </g> \
+                    <g layout-css="flex: 1; flexDirection: row"> \
+                        <g class="y-axis left"> \
+                            <g class="label-container"> \
+                                <g layout-css="height: 0; width: 0"> \
+                                    <text class="label" transform="rotate(-90)"/> \
+                                </g> \
+                            </g> \
+                            <g class="padding"/> \
+                            <g class="axis" layout-css="width: 0"/> \
+                        </g> \
+                        <svg class="plot-area" layout-css="flex: 1"/> \
+                        <g class="y-axis right"> \
+                            <g class="axis" layout-css="width: 0"/> \
+                            <g class="padding"/> \
+                            <g class="label-container"> \
+                                <g layout-css="height: 0; width: 0"> \
+                                    <text class="label" transform="rotate(-90)"/> \
+                                </g> \
+                            </g> \
+                        </g> \
+                    </g> \
+                    <g class="x-axis"> \
+                        <g class="axis"/> \
+                        <g class="label-container"> \
+                            <text class="label" layout-css="height: 0; width: 0"/> \
+                        </g> \
+                    </g>');
+
+                // configure the y axis containers based on y-axis orientation
+                var opposingOrientation = yAxis.orient() === 'left' ? 'right' : 'left';
```

Comment:
There are a few statically defined values in the `layout` argument object. I assume that's because `layout` doesn't support applying deltas using the object syntax? Is that something we should create an issue for?